structs/structs.go

```diff
@@ -7,16 +7,16 @@ type Input struct {
 }
 
 type Schedule struct {
-	FallCourses   []Course `json:"fallTermCourses"`
-	SpringCourses []Course `json:"springTermCourses"`
-	SummerCourses []Course `json:"summerTermCourses"`
+	FallCourses   []Course `json:"fallCourses"`
+	SpringCourses []Course `json:"springCourses"`
+	SummerCourses []Course `json:"summerCourses"`
 }
 
 type Assignment struct {
 	StartDate string  `json:"startDate"`
 	EndDate   string  `json:"endDate"`
 	BeginTime string  `json:"beginTime"`
-	EndTime   string  `json:"endtime"`
+	EndTime   string  `json:"endTime"`
 	HoursWeek float32 `json:"hoursWeek"`
```

Comment:
The swagger document lists this as `integer`. Not a huge deal on our end (or probs other alg as JavaScript treats them the same) but just noting.

Reply:
We were working with strings for the beginTime and endTime as then we can have the leading 0 for times like 0830 etc. for formatting

Reply:
I'm talking about `HoursWeek`. Don't think I highlighted the right field.